docs/effects/pixel-mapper.md

```diff
@@ -220,8 +220,9 @@ Pixel Mapper Layer Masters
 
 You can assign each of the four layers to a master. The master allows
 you to adjust the layer controls in real time. When the master is
-assigned to a fader handle using the \[Assign Masters\], \[Pixel Mapper\]
-controls, the fader will control opacity of the layer.
+assigned to a fader handle using \<Record\>, \[Create Master\], \[Pixel Mapper\],
+selecting \[Layer 1\] through \[Layer 4\] and finally pressing a handle select key,
```

Comment:
Did we settle on if we were referring to it as select/Select/**Select** - doesn't make too much odds, these changes all look good

Reply:
Honestly, for the moment, I decided to leave this up to Tim and Fraser so that this is done by _one_ hand. This is why I keep reporting other stuff I deem worth having a second glance on, and leave the swop/select/blue faderkey thing untouched. I hope that's okay.

Reply:
I think "Select" (capital not bold)

It would be a good idea to have a style guide written for the manual so that things like this can be consistent